plutus-use-cases/test/Spec/Crowdfunding.hs

```diff
@@ -59,7 +59,9 @@ contrib3 = contrib ds (Wallet 3) where
 
 -- | Collect the contributions of a crowdfunding campaign
 collect :: Wallet -> CampaignPLC -> [(TxOutRef', Wallet, UTXO.Value)] -> Trace [Tx]
-collect w c contributions = walletAction w $ CF.collect c ins where
+collect w c contributions =
+  fmap fst . walletAction w $ CF.collect c ins
```

Comment:
Easier to read as `fst <$> walletAction w $ ...` if precedence works with that.

Reply:
yeah precedence meant it didn't work. reverting anyway so it doesn't matter